backend/services/api-test/src/main/java/io/metersphere/api/service/ApiFileResourceService.java

```diff
@@ -331,6 +388,15 @@ public List<ApiFileResource> selectByApiScenarioId(List<String> scenarioIds) {
         return apiFileResourceMapper.selectByExample(example);
     }
 
+    public List<ApiFileResource> selectByResourceIdAndFileIds(String resourceId, List<String> fileIds) {
+        if (StringUtils.isBlank(resourceId) || CollectionUtils.isEmpty(fileIds)) {
+            return new ArrayList<>();
+        }
+        ApiFileResourceExample example = new ApiFileResourceExample();
+        example.createCriteria().andResourceIdEqualTo(resourceId).andFileIdIn(fileIds);
+        return apiFileResourceMapper.selectByExample(example);
+    }
+
     public void batchInsert(List<ApiFileResource> insertApiFileResourceList) {
         apiFileResourceMapper.batchInsert(insertApiFileResourceList);
     }
```

Comment:
Code doesn't make sense after changes made. Please provide detailed code or specific parts of the code to analyze.

Also, there might be some issues with API usage/structure since it looks like you've replaced certain keywords which don't seem related with their old versions.

Please correct your code and ask here again if need advice on how to improve it. Thanks!